### ISSUE TITLE
Source of the datasets

### ISSUE THREAD

**Issue by fmonti** [launchnlp/LitCab#4]
Hey!
Thank you for the nice work! I'm reading the paper and trying to get an understanding of where the data you used come from. Focusing in particular on the NQ dataset (which is the main one I currently looked at), can you provide more details on how you got the split between training and test set that you used? NQ consists of "307,373 training examples with single annotations, 7,830 examples with 5-way annotations for development data, and a further 7,842 examples 5-way annotated sequestered as test data" (https://research.google/pubs/natural-questions-a-benchmark-for-question-answering-research/), but these numbers are not consistent with what you used in your dataset.

I thought you then used the "open" version of NQ (https://github.com/google-research-datasets/natural-questions/tree/master/nq_open), however the questions don't appear to match (and the training set contained in this repository is smaller than what provided as part of NQ-open). Can you provide some details on this?

I also don't understand why you used only 1000 questions for evaluation from your own test set, when this contains 3610 question. Is there a particular reason why you downsampled the test set?

Thank you for your help!
